docker-compose.yml

```diff
@@ -15,11 +54,28 @@ services:
       - "9200:9200"
 
   web:
-    build: .
+    build:
+      context: .
+      dockerfile: Dockerfiles/Dockerfile.app
     expose:
       - "5000"
     ports:
       - "5000:5000"
+    environment:
+      ELASTICSEARCH_URL: search:9200
```

Comment:
I added this line because I think we need to specify the host `search` in the docker-compose instead of default localhost